service-tests/microbadger.js

```diff
@@ -2,13 +2,14 @@
 
 const Joi = require('joi');
 const ServiceTester = require('./runner/service-tester');
-const { isFileSize } = require('./helpers/validators')
+const { isFileSize } = require('./helpers/validators');
+const { invalidJSON } = require('./helpers/response-fixtures');
 
-const t = new ServiceTester({ id: 'microbadger', title: 'MicroBadger' })
+const t = new ServiceTester({ id: 'microbadger', title: 'MicroBadger' });
 module.exports = t;
 
 t.create('image size without a specified tag')
-  .get('/image-size/_/hello-world.json')
+  .get('/image-size/_/centos.json')
```

Comment:
This is a quick fix. For `hello-world` badge returned `error`, because `image.DownloadSize` is `undefined` in
https://github.com/badges/shields/blob/8ad176d014bdddc95f06a95ae4ff82c7697bb569/server.js#L6479
 @goozler Do you have any idea why it changed?
There is no `DownloadSize` property in main object but there is `"DownloadSize": 0` in `Versions[0]`. We can replace `error` in this case with `0`, but I'm not sure it's the best solution (and I would like to fix this in a separate PR). 
```
curl "https://api.microbadger.com/v1/images/library/hello-world" -s | jq .
{
  "LatestSHA": "690d802025313dfb6400e3872bcbf3ab5e486c5160b89fa0b3edf0332bfa4cf1",
  "WebhookURL": "https://hooks.microbadger.com/images/library/hello-world/6Yx3YymZyiVymVHeR6XIH_Yzy2s=",
  "UpdatedAt": "2018-01-28T12:50:00.104461Z",
  "Description": "Hello World! (an example of minimal Dockerization)",
  "LastUpdated": "2018-01-08T20:43:23.336375Z",
  "PullCount": 128583318,
  "StarCount": 431,
  "Versions": [
    {
      "SHA": "690d802025313dfb6400e3872bcbf3ab5e486c5160b89fa0b3edf0332bfa4cf1",
      "Tags": [
        {
          "tag": "linux"
        },
        {
          "tag": "latest"
        }
      ],
      "ImageName": "hello-world",
      "Author": "",
      "LayerCount": 2,
      "DownloadSize": 0,
      "Created": "2017-11-21T00:23:18.797568Z"
    }
  ],
  "Id": "690d802025313dfb6400e3872bcbf3ab5e486c5160b89fa0b3edf0332bfa4cf1",
  "ImageName": "hello-world",
  "ImageURL": "https://hub.docker.com/r/library/hello-world/",
  "LayerCount": 2,
  "LatestVersion": "linux"
}
```

Reply:
I suppose it could be Docker or Microbadger related issue. They had some issues with their service and the support suggested to make a POST request to the WebhookURL to recalculate statistics. I think the error message in that case is fine.

I have my own docker hub image and the size of it was reset several times but I didn't change anything. I called the webhook url manually and it solved the issue temporarily. Last two months everything has been fine, so, I assume they fixed it. But for some images the problem still exists.

I think the size may be broken after new Docker version is released or something like that. 

I agree with changing the image for tests if it's broken. That makes sense for me. Also on the demo page the `httpd` image doesn't have the size too and the webhook url doesn't help and it could be changed too, I guess.

I chose `hello-world` for tests because I thought it wouldn't be changed and it should work all the time :-)

In general, I don't know how to solve it because it's on the Microbadger side.